doc/source/avl_ludwigsburg_de.md

```diff
@@ -0,0 +1,151 @@
+# AVL Ludwigsburg
+
+Support for schedules provided by [avl-ludwigsburg.de](https://www.avl-ludwigsburg.de) located in Baden Württemberg, Germany.
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: avl_ludwigsburg
+      args:
+        city: Ludwigsburg
+        street: Bahnhofstraße
+```
+
+### Configuration Variables
+
+**city**<br>
+*(string) (required)*
+
+**street**<br>
+*(string) (optional - depending on city)*
+
+## Example
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: avl_ludwigsburg
+      args:
+        street: Bahnhofstraße
+        city: Ludwigsburg
+      customize:
+        - type: LVP 4-Rad
+          alias: Gelber Sack 4-Rad
+          show: true
+        - type: LVP
+          alias: Gelber Sack
+          show: true
+        - type: Papier
+          alias: Papier
+          show: true
+        - type: Papier 4-Rad
+          alias: Papier 4-Rad
+          show: true
+        - type: Restmüll 4-Rad
+          alias: Restmüll 4-Rad
+          show: true
+        - type: Restmüll
+          alias: Restmüll
+          show: true
+        - type: Biomüll
+          alias: Biomüll
+          show: true
+        - type: Glas
+          alias: Glas
+          show: true
+```
+
+Use `sources.customize` to filter or rename the waste types:
```

Comment:
Is there a difference in the 2nd example compared to the first? If not, please remove it.

Reply:
Fixed with 99f76ee391ace6bda6a79fd79bc9672f1e57c4a8